File: Python/secao08/Exercicio04.py
# var
from typing import SupportsComplex


vetor = []
soma = 0

# processing
for i in range(0, 20):
    num = int(input("Informe um valor para o vetor: "))
    vetor.append(num)
    soma = soma + num

print("A soma de todos os valores é: {0}".format(soma))

File: Java/src/secao08/Exercicio04.java
package secao08;

import java.util.Scanner;

public class Exercicio04 {
    public static void main(String[] args) {
        // var
        int soma = 0;
        int[] vetor = new int[20];
        Scanner teclado = new Scanner(System.in);

        for (int i = 0; i < vetor.length; i++) {
            // input
            System.out.printf("Digite um valor (%d/20): ", (i + 1));
            vetor[i] = teclado.nextInt();

            soma = soma + vetor[i];
        }
        System.out.println("A soma de todos os elementos é: " + soma);
        teclado.close();
    }
}
